lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -139,83 +132,76 @@ abstract class SqlParserTestBase : TestBase() {
+        val transformedActualExprNode = MetaStrippingRewriter.stripMetas(transformedActualStatement.toExprNode(ion))
+        assertEquals(MetaStrippingRewriter.stripMetas(actualExprNode), transformedActualExprNode)
     }
 
-    private fun assertRoundTripIonElementToPartiQlAst(parsedElement: SexpElement, expectedSexpAst: SexpElement) {
-        // #1 We can transform the parsed PartiqlAst element.
-        val transformedParsedElement = PartiqlAst.transform(parsedElement)
+    private fun assertRoundTripIonElementToPartiQlAst(actualElement: SexpElement, expectedElement: SexpElement) {
+        // #1 We can transform the actual PartiqlAst element.
+        val transformedActualElement = PartiqlAst.transform(actualElement)
 
         // #2 We can transform the expected PartiqlAst element.
-        val transformedExpectedElement = PartiqlAst.transform(expectedSexpAst)
+        val transformedExpectedElement = PartiqlAst.transform(expectedElement)
 
         // #3 The results of both transformations match.
-        assertEquals(transformedExpectedElement, transformedParsedElement)
+        assertEquals(transformedExpectedElement, transformedActualElement)
 
-        // #4 Re-transforming the parsed PartiqlAst element and check if it matches the expected AST.
-        val reserializedAst = transformedParsedElement.toIonElement()
-        assertEquals(expectedSexpAst, reserializedAst)
+        // #4 Re-transforming the actual PartiqlAst element and check if it matches the expected AST.
+        val reserializedActualElement = transformedActualElement.toIonElement()
+        assertEquals(expectedElement, reserializedActualElement)
 
         // #5 Re-serializing the expected PartiqlAst element matches the expected AST.
         // Note:  because of #3 above, no need for #5.
     }
 
     /**
-     * Strips metas from the [parsedExprNode] so they are not included in equivalence checks
-     * and round-trip the resulting [parsedExprNode] AST through [toAstStatement] and [toExprNode].
+     * Strips metas from the [actualExprNode] so they are not included in equivalence checks
+     * and round-trip the resulting [actualExprNode] AST through [toAstStatement] and [toExprNode].
      *
      * Verify that the result matches the original without metas.
      */
-    private fun pigExprNodeTransformAsserts(parsedExprNode: ExprNode) {
-        val parsedExprNodeNoMetas = MetaStrippingRewriter.stripMetas(parsedExprNode)
-        val statement = parsedExprNodeNoMetas.toAstStatement()
-        val exprNode2 = statement.toExprNode(ion)
-        assertEquals(parsedExprNodeNoMetas, exprNode2)
+    private fun pigExprNodeTransformAsserts(actualExprNode: ExprNode) {
+        val actualExprNodeNoMetas = MetaStrippingRewriter.stripMetas(actualExprNode)
+        val actualStatement = actualExprNodeNoMetas.toAstStatement()
+        val transformedActualExprNode = actualStatement.toExprNode(ion)
```

Comment:
This is not really a `transfromedExprNode`. `PartiqlAst.transform` converts `IonSexp` to `PartiQLAst`. In some sense, `transform` is `deserialize` and here we are going from `PIG AST -> ExprNode`. I would rename this to `roundTrippedExprNode`.